docs/contributors/document.md

```diff
@@ -63,6 +63,29 @@ To create links that work in all contexts, you should use absolute path links wi
 
 This way they will be properly handled in all three aforementioned contexts.
 
+### Code Examples
+
+The code example in markdown should be wrapped in three tick marks <code>```</code> and can additionally include a language specifier. See this [Github documentation around fenced code blocks](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks).
+
+A unique feature to the Gutenberg documentation is the `codetabs` toggle, this allows two versions of code to be shown at once. This is used for showing both `ESNext` and `ES5` code samples. For example, [on this block tutorial page](/docs/designers-developers/developers/tutorials/block-tutorial/block-controls-toolbar-and-sidebar.md).
+
+Here is an example `codetabs` section:
+
+		{% codetabs %}
+		{% ESNext %}
+		```js
+			// ESNext code here
+		```
+		{% ES5 %}
+		```js
+			// ES5 code here
+		```
+		{% end %}
```

Comment:
Was there a reason for using two tabs? Usually one tab (four spaces) is enough to create a code block.

Reply:
Overly, aggressive tabbing. :-)